src/fake/auth.js

```diff
@@ -1,8 +1,19 @@
 import _ from 'lodash';
+import jwt from 'jsonwebtoken';
 
 const parseFormData = data =>
   _.fromPairs(data.split('&').map(keyValue => keyValue.split('=').map(decodeURIComponent)));
 
+const hostnameFromToken = token => {
+  let unsigned;
+  try {
+    unsigned = jwt.verify(token, 'valid-secret');
```

Comment:
Do we need the unsigned variable? We could just do `return jwt.verify(token, 'valid-secret').hostname`.